docs/howto/manage-web-app-charms/configure-web-app-charm.rst

```diff
@@ -85,8 +85,132 @@ The configuration can be set on the deployed charm using:
 
     juju config <app name> token=<token>
 
-Manage secrets for a 12-factor app charm
-----------------------------------------
+Add a custom action
+-------------------
+
+For certain commands or processes that you will run many times, you can set up
+custom Juju actions to expedite your workflow. Actions are handled in the Ops
+library, and Juju manages the action's workflow via tasks and operations. To
+implement a custom action, you need to declare the action in
+``charmcraft.yaml`` and define an event handler into the Ops framework of your
+charm.
+
+.. seealso::
+
+    :external+ops:ref:`Ops | Manage actions <manage-actions>`
+
+The custom action must be defined in ``charmcraft.yaml`` under the
```

Comment:
```suggestion
The custom action must be defined in the project file under the
```

I suppose it's time to bring this phrase up. It's the generic phrase for all `*craft.yaml` files in our products, and reduces the amount of code in the text.

Kindly consider it for all instances.